### DISCUSSION TITLE
Ability to bind system keys (Ctrl+Alt+Tab)

### DISCUSSION THREAD

**Discussion by crossatko** [Schneegans/Fly-Pie#135] [Ideas & Feature Discussions]
Hi,
it would be really awesome to be able to use system keys as shortcut triggers.
For example MX Master mouse has a thumb button that sends Ctrl+Alt+Tab key combination.

This extension would be really awesome to overcome the lack of drivers for this mouse for gestures like swiping to change desktop or quickly maximize or minimize window.

Have a great day!

**Comment by Schneegans** [comment]
By design, Fly-Pie does not let you choose shortcuts for menus which are already used by the system. You won't be able to bind a menu to <kbd>print</kbd> for example, if this is used to capture screenshots, for example. However, you can make the shortcut actions simulate these shortcuts. So you _cannot_ select <kbd>print</kbd> as a trigger for a menu, but you _can_ select <kbd>print</kbd> for shortcut actions.

Nevertheless, I can reproduce your issue. If I disable Ctrl+Alt+Tab in the system-wide shortcuts, I can still not make it open a menu. After playing around a bit, it seems that the <kbd>Tab</kbd> key is somewhat special. I cannot make it open a menu with any modifier key. Maybe that's a bug, or it's a limitation as this key may get consumed by GNOME Shell.

I'll have to investigate :)

As it may be a bug in Fly-Pie, I'll transfer this to be an issue. Thanks for the report!

**Reply by Schneegans** [reply]
Let's continue over there: #137
